app/assets/javascripts/discourse/app/components/iframed-html.gjs

```diff
@@ -0,0 +1,24 @@
+import Component from "@glimmer/component";
+import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import concatClass from "discourse/helpers/concat-class";
+import { bind } from "discourse-common/utils/decorators";
+
+export default class IframedHtml extends Component {
+  @bind
+  writeHtml(element) {
+    const iframeDoc = element.contentWindow.document;
+    iframeDoc.open("text/html", "replace");
+    iframeDoc.write(this.args.html);
+    iframeDoc.close();
+  }
+
+  <template>
+    {{! template-lint-disable require-iframe-title }}
+    <iframe
+      {{didInsert this.writeHtml}}
```

Comment:
I think we'll want `didUpdate` as well? Previously it was using the classic `didRender`, which fires on initial render and subsequent re-renders.